nix-action.yml.mustache

```diff
@@ -1,39 +1,151 @@
 # This file was generated from `meta.yml`, please do not edit manually.
 # Follow the instructions on https://github.com/coq-community/templates to regenerate.
+{{!# From now on we switch template variable syntax to <%X%>}}
+{{=<% %>=}}
 name: Nix CI
 
 on:
   push:
     branches:
-      - {{branch}}{{^branch}}master{{/branch}}
+      - <%branch%><%^branch%>master<%/branch%>
   pull_request:
     branches:
       - '**'
 
 jobs:
-  build:
+  setup:
     runs-on: ubuntu-latest
+<%^ nix-tasks%>
+    outputs:
+      task: ${{ steps.set-matrix.outputs.task }}
+<%/ nix-tasks%>
+    steps:
+    - name: Do nothing
+      run: echo "No setup, using meta.yml tasks"
```

Comment:
Nop. Because mustache has no way to branch over list emptyness, and github action won't accept an empty task list, so there has to be an element in it.

Reply:
But isn't the solution to have a default task if none is specified? Or to just not generate the GitHub Action file in this case?

Reply:
We discussed this in a call.